benchmark/Dockerfile

```diff
@@ -3,9 +3,20 @@ FROM ubuntu:22.04
 ENV CSV_DIR /csv
 ENV SERIALIZED_DIR /serialized
 ENV DEBIAN_FRONTEND=noninteractive
-RUN apt-get update && apt-get install -y --no-install-recommends apt-utils curl ca-certificates apt-transport-https gnupg software-properties-common
-RUN curl -fsSL https://deb.nodesource.com/setup_20.x | bash -
-RUN apt-get update && apt-get -y install python3-dev python3-pip python-is-python3 cmake nodejs jq curl git libssl-dev libcurl4-openssl-dev
+RUN apt-get update && apt-get -y install \
+    build-essential \
+    ccache \
+    cmake \
+    curl \
+    git \
+    jq \
+    libssl-dev \
+    libcurl4-openssl-dev \
+    ninja-build \
+    python3-dev \
+    python3-pip \
+    python-is-python3
+
```

Comment:
It sort of looks like we forgot to install npm... https://github.com/kuzudb/kuzu/actions/runs/6655831578/job/18087424945?pr=2273 I have a failure here.

Reply:
Nevermind, that's because benchmark no longer needs npm. We should merge this soon then so CI isn't broken for too long.